code_examples/sdk_examples/src/dapp/verifier/verifyCredentialMessage.ts

```diff
@@ -64,9 +64,13 @@ export async function main({ session, verifierKeys }: Param) {
       attestationChain,
       credential.rootHash
     )
+    if (attestation.revoked) {
+      console.log("Credential has been revoked and hence it's not valid.")
+      return
```

Comment:
It's not your code but maybe we should throw here.

Reply:
https://github.com/KILTprotocol/docs/pull/201/commits/d63cff7b0b3a61dfb01914c3262e3ff854216082